.circleci/config.templ.yml

```diff
@@ -1015,7 +1015,7 @@ jobs:
     steps:
       - run_test:
           wait: mysql
-          pattern: 'pymysql$'
+          pattern: 'mysql'
```

Comment:
This seems to duplicate some test runs from the `mysqldb` and `mysql_connector` suites. Maybe delete those other circleci suites?

Reply:
Ive removed these changes